### DISCUSSION TITLE
运行微调后的模型出现torch.size不匹配问题

### DISCUSSION THREAD

**Discussion by KYW260129** [zai-org/ChatGLM3#1010] [Q&A]
<img width="828" alt="图片1" src="https://github.com/THUDM/ChatGLM3/assets/95842335/45d0fbf4-39f5-46ec-904e-78460dee8cf8">
怎么解决大佬们，推理没有问题：
<img width="877" alt="图片2" src="https://github.com/THUDM/ChatGLM3/assets/95842335/8a87906a-2665-4299-8c04-c399a545a9d3">

**Comment by wks408451747** [comment]
解决了吗？我也遇到了同样的问题！！！

**Reply by KYW260129** [reply]
https://zhuanlan.zhihu.com/p/681326841你好，我解决了，我采用了链接中第二种合并模型的方法
